File: spring-java/src/main/java/com/jmpark/springjava/entity/ParkingArea.java
package com.jmpark.springjava.entity;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.ToString;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import java.math.BigDecimal;

@Entity
@RequiredArgsConstructor
@Getter
@ToString
@EqualsAndHashCode(callSuper = false)
public class ParkingArea extends BaseTimeEntity {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    Integer id = 0;

    String districtId = null;
    String code = null;
    String name = null;
    String type = null;
    String category = null;
    Integer segment = 0;
    String level = null;
    Integer isRotation = 0;
    Integer isPay = 0;
    String memo = null;
    Integer managementId = 0;
    BigDecimal latitude = BigDecimal.ZERO;
    BigDecimal longitude = BigDecimal.ZERO;
    String lastUpdated = null;
}

File: spring-kotlin/src/main/kotlin/com/jmpark/springkotlin/entity/ParkingArea.kt
package com.jmpark.springkotlin.entity

import java.math.BigDecimal
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType
import javax.persistence.Id

@Entity
data class ParkingArea (
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    val id : Int = 0,

    var districtId: String? = null,
    var code: String? = null,
    var name: String? = null,
    var type: String? = null,
    var category: String? = null,
    var segment: Int? = 0,
    var level: String? = null,
    var isRotation: Int? = 0,
    var isPay: Int? = 0,
    var memo: String? = null,
    var managementId: Int? = 0,
    var latitude: BigDecimal? = BigDecimal.ZERO,
    var longitude: BigDecimal? = BigDecimal.ZERO,
    var lastUpdated: String? = null
) : BaseTimeEntity()